actors.md

```diff
@@ -455,6 +455,12 @@ type StorageMinerActorState struct {
 	## when a PoSt is submitted (not as each new sector commitment is added).
     provingSet &SectorSet
 
+    ## Faulty sectors reported since last SubmitPost, up to the current PoSt challenge time.
+    currentFaultSet BitField
+
+    ## Faults for the next round of PoSt.
```

Comment:
We can be more clear: "Faults submitted after the current proving period's challenge time but before the PoSt for that period is submitted. These become the `currentFaultSet` when a PoSt is submitted."

These fields are still a bit hard to describe in the period between when a SubmitPoSt lands and when the subsequent proving period opens, but I think we have the same idea.